ext/ext.md

```diff
@@ -31,14 +31,16 @@ Content-Type: application/json
 
 {
     "extensions: [
-        "_<ns>/<ext>/<component>",
+        {
```

Comment:
Curious why to have a hashmap inside an array here?

Reply:
We updated the response to be an array of extension objects rather than just the strings. This will enable us to add more details about an extension in the response if needed in the future. /cc @sudo-bmitch

Reply:
ah ok. might be good for me to include a basic golang source with the structs for easy marshalling of this.